src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -700,6 +700,7 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
                     }
                 };
 
+                // TODO, register the new event handlers to capture new data here so that we can stuff them in TraceGC
```

Comment:
>// TODO, register the new event handlers to capture new data here so that we can stuff them in TraceGC [](http://example.com/codeflow?start=16&length=102)

what are these new event handlers? also are you planning to do this soon? if so you probably don't need to add a comment here.